webserver/main.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/gin-gonic/gin"
+	"gopkg.in/yaml.v2"
+)
+
+type Config struct {
+	Server struct {
+		Port string `yaml:"port"`
+		Host string `yaml:"host"`
+	} `yaml:"server"`
+	Database struct {
+		Username string `yaml:"user"`
+		Password string `yaml:"pass"`
+	} `yaml:"database"`
+}
+
+type Wedstrijd struct {
+	Datum         string   `json:"Datum"`
+	Plaats        string   `json:"Plaats"`
+	LSR           bool     `json:"LSR"`
+	MSR           bool     `json:"MSR"`
+	KSR           bool     `json:"KSR"`
+	Jeugd         bool     `json:"Jeugd"`
+	BSR           bool     `json:"BSR"`
+	Kwalificatie  bool     `json:"Kwalificatie"`
+	Afstanden     []string `json:"Afstanden"`
+	MinLeeftijd   string   `json:"min.leeftijd"`
+	Organisator   []string `json:"Organisator"`
+	Inschrijflink []string `json:"Inschrijflink"`
```

Comment:
I see that go doesn't have a native URL datatype, but the naming could be more consistent to indicate that the "Uitslag" variable is also a link/url